index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute page</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div id="main">
+        <h1 id="title" class="name">Andre Acosta</h1><br>
+        <p>The Best Coding Partner</p><br>
+        <div id="img-div">
+            <img id="img" class="responsive" src="https://www.oficinadanet.com.br/imagens/post/25092/cc01.jpg" alt="Hero Image">
+            <p id="img-caption">The TimeLine of Andre's Life</p>
+        </div>
```

Comment:
To write code accordingly with best practices, we should always use semantic tags in HTML. It also helps with SEO.